backend/transcribee_backend/routers/document.py

```diff
@@ -191,3 +204,47 @@ async def websocket_endpoint(
         document=document, websocket=websocket, session=session
     )
     await connection.run()
+
+
+@document_router.post("/{document_id}/add_media_file/")
+def add_media_file(
+    document_id: uuid.UUID,
+    tags: list[str] = Form(),
+    file: UploadFile = File(),
+    session: Session = Depends(get_session),
+    authorized_worker: Worker = Depends(get_authorized_worker),
+) -> ApiDocument:
+    statement = select(Task).where(
+        Task.document_id == document_id, Task.task_type == TaskType.REENCODE
+    )
+    task = session.exec(statement).one_or_none()
+    if task is None:
+        raise HTTPException(status_code=403)
+
+    if task.assigned_worker != authorized_worker:
+        raise HTTPException(status_code=403)
+
+    stored_file = media_storage.store_file(file.file)
+    file.file.seek(0)
+
+    media_file = DocumentMediaFile(
+        created_at=now_tz_aware(),
+        changed_at=now_tz_aware(),
+        document_id=task.document_id,
+        file=stored_file,
+        content_type=magic.from_descriptor(file.file.fileno(), mime=True),
+    )
+
+    session.add(media_file)
+    session.commit()
+    session.refresh(media_file)
```

Comment:
This refresh is also unnecessary. Doesn't `sqlmodel` automatically expire on commit and then fetch the most recent data when its used later?

Reply:
Nice, thanks for all the three comments. I think I just commited the `add`/`commit`/`refresh`-code somewhere from the sqlmodel docs without thinking about it again